### PR TITLE
Help Requests form

### PR DESCRIPTION
**Pull request by marianbme**

### CODE REVIEW

File: StreetCare/StreetCare/Views/Screens/AddHelpRequestForm.swift [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+            Text(title).font(.headline)
+            
+            // TextField with border
+            TextField(placeholder, text: Binding(
+                get: { text },
+                set: { text = String($0.prefix(limit)) }
+            ))
+            .frame(height:50)
+            .textFieldStyle(PlainTextFieldStyle()) // Use PlainTextFieldStyle to customize the border
+            .padding() // Padding inside the TextField for spacing
```
Comment (phillipsk) [inline StreetCare/StreetCare/Views/Screens/AddHelpRequestForm.swift:250]: I noticed that there are a lot of comments throughout the code. While comments can be helpful for explaining complex logic, having too many comments might reduce readability and clutter the code. Ideally, the code itself should be self-explanatory where possible (through clear variable names, functions, and structure).

Perhaps we could remove or rephrase comments that are redundant or overly detailed, keeping only those that add real value or explain non-obvious logic.

**Review by phillipsk — CHANGES_REQUESTED** [review]

**Comment by phillipsk** [comment]
Please add a better title for this PR.

**Comment by marianbme** [comment]
fixed the requested changes, please check and let me know

File: StreetCare/StreetCare/Views/Screens/AddHelpRequestForm.swift [hunk 2]
```diff
@@ -189,43 +191,63 @@ struct FormButtons: View {
...
                 .fontWeight(.semibold)
+                
 
             Text("If this homeless person requires further assistance, please provide additional details so we can share with the community and rally support to help them.")
                 .font(.subheadline)
                 .foregroundColor(.gray)
         }
         .frame(maxWidth: .infinity, alignment: .leading)
-        .padding(.top, 0) // Remove or reduce top padding
+        .padding(.top, -32)
```
Comment (phillipsk) [inline StreetCare/StreetCare/Views/Screens/AddHelpRequestForm.swift:233]: change padding to        ` .padding(.top, 12)`

**Review by phillipsk — CHANGES_REQUESTED** [review]

**Review by phillipsk — APPROVED** [review]

**Pull request merged**